### ISSUE TITLE
Navigation of stories is inconvenient/small in the mobile version

### ISSUE THREAD

**Issue by froseman-codex** [ookami-kb/storybook_flutter#69]
Hello.
It takes a long time to scroll to the desired story.
version 0.10.0

Thanks)

<img src="https://user-images.githubusercontent.com/96050737/150120767-fd3b686e-6eee-4be4-8466-f38ef33ef29f.png" width="250"/>

**Comment by ilexbor-codex** [comment]
Support
Navigating the Story in the previous version was more convenient because more Story was displayed on the screen than in the current latest version

**Comment by ookami-kb** [comment]
Makes sense. Let me think about how it would be better to address that, probably providing an option to use drawer, as it was in the previous version.

But keep in mind, that storybook is probably not the best to be launched on mobile – you can use a bigger screen like tablet/desktop where you can turn on the option to display the contents in the side panel. With device preview, you can still how your layout will look like on smaller screens.

**Comment by ilexbor-codex** [comment]
@ookami-kb
We use storybook to test our widgets by both developers and testers, so it is important for us to be able to run storybook on mobile devices.

**Comment by searchy2** [comment]
Running storyboard on mobile is helpful with testing how components work with touch gestures on mobile. Would be great to have the drawer back!

**Comment by ookami-kb** [comment]
Yes, that makes sense. Most probably, it should be done with a plugin. I'll try to come up with something in the neat future.
